src/main/java/com/ita/if103java/ims/exception/BaseRuntimeException.java

```diff
@@ -2,22 +2,10 @@
 
 public class BaseRuntimeException extends RuntimeException {
 
-    private String name;
-
     BaseRuntimeException(String message){
         super(message);
     }
 
-    BaseRuntimeException(String message, String name){
-        super(message);
-        this.name = name;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
+    BaseRuntimeException() {
     }
 }
```

Comment:
Lets add additional constructors for our `BaseRuntimeException` class. The same for `CRUDException` and `UserNotFoundException`.
```java
public BaseRuntimeException() {
        super();
    }

    public BaseRuntimeException(String message, Throwable cause) {
        super(message, cause);
    }

    public BaseRuntimeException(Throwable cause) {
        super(cause);
    }
```
Something like this. So we can to pass our exception to constructor and see the actual error in log stacktrace:
```java
throw new CRUDException("Some info here", e);
```